src/schema/v2/artwork/utilities.ts

```diff
@@ -125,3 +126,19 @@ export const isEligibleForOnPlatformTransaction = ({
 
   return false
 }
+
+export const isEligibleToCreateAlert = (artwork: {
+  artists: { id: string }[]
+  category: string | null
+}) => {
+  const { artists, category } = artwork
+
+  // an alert must be associated with at least one valid artist
+  if (artists.length == 0) return false
+
+  // an alert should be associated with a real medium, and not just "Other"
+  if (!category) return false
+  if (category == mediums.Other.name) return false
```

Comment:
So far our logic has only tested for the presence of artists (i.e. L137)

This adds an opinionated test for the presence of a valid medium as well (L140–141), on the reasoning that…

- A null medium would likely result in overly broad alerts
- A catch-all "Other" medium would not generate a useful alert, and presumably [not even a valid one](https://github.com/artsy/metaphysics/blob/89e7f5b1821136db52cb0a32bb111f79f8bd9005/src/lib/artworkMediums.ts#L91)

Reply:
> A null medium would likely result in overly broad alerts

But it is still possible to create and alert without it, from an artist page, for instance. I guess I'm ok with adding such check here, but wondering if this mismatch in validations can cause problems in future

Reply:
>  still possible to create and alert without it, from an artist page

Yep, and I expect this flag would not be consulted in the case of setting an alert from the artist artwork grid. This should only govern the "alert-ability" of an individual artwork, where I think these tighter rules make sense.